sql/moz-fx-data-shared-prod/search_derived/mobile_search_aggregates_for_searchreport_v1/query.sql

```diff
@@ -41,8 +41,8 @@ SELECT
 FROM
   search.mobile_search_clients_engines_sources_daily
 WHERE
-  app_name IN ('Fenix', 'Fennec', 'Firefox Preview', 'Focus')
-  AND app_name IS NOT NULL
+  normalized_app_name IN ('Fenix', 'Fennec', 'Firefox Preview', 'Focus')
+  AND normalized_app_name IS NOT NULL
```

Comment:
```suggestion
  normalized_app_name IN ('Fenix', 'Fennec', 'Firefox Preview', 'Focus')
```

The `IS NOT NULL` is redundant, no?